src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -34,6 +35,9 @@ class ContainerAwareEventManager extends EventManager
     private $methods = [];
     private $container;
 
+    /**
+     * @param list<string|EventSubscriber|array{string[], string|object}> $subscriberIds list of subscriber, subscriberIds or couple of events+eventListener
```

Comment:
`List of subscribers, subscriber ids, or [events, listener] tuples`

Reply:
Should be changed to `[events, listener id] tuples` btw, no need to handle the case where this an object - isn't it?

Reply:
I just realized how this works, forget about my comments here :)